### PR TITLE
Add `linux-dev`

### PR DESCRIPTION
**Pull request by Arignir**
This wil be the build manifest for the whole linux kernel, but only the development version is enabled as of now.

The whole kernel will be enabled at a later stage.

### CODE REVIEW

File: kernel/linux/linux.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    boot_dir = os.path.join(build.install_cache, 'boot')
+    modules_dir = os.path.join(build.install_cache, 'usr')
+    include_dir = os.path.join(build.install_cache, 'usr')
+
+    os.makedirs(boot_dir, exist_ok=True)
+    os.makedirs(modules_dir, exist_ok=True)
+    os.makedirs(include_dir, exist_ok=True)
+
+    # make(f'INSTALL_PATH={boot_dir}', 'install')
```
Comment (doom) [inline kernel/linux/linux.py:25]: Should these commented lines be removed ?

File: kernel/linux/linux.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    )
+
+    # Drain documentation
+    # packages['kernel/linux-doc'].drain_build_cache('Documentation/*', 'usr/doc/linux/')
+
+    # Remove the dependencie of `kernel/linux-dev` to `kernel/linux`
+    packages['kernel/linux-dev'].run_dependencies = dict()
+
+    # Change the description of kernel/linux-dev
+    packages['kernel/linux-dev'].description = 'Headers to compile or write a software based on the Linux kernel.'
```
Comment (doom) [inline kernel/linux/linux.py]:
```suggestion
    packages['kernel/linux-dev'].description = 'Headers to compile or write software based on the Linux kernel.'
```

File: kernel/linux/linux.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    # Packages member of `raven-os/essentials` should explicitely state all
```
Comment (doom) [inline kernel/linux/linux.py]:
```suggestion
    # Packages member of `raven-os/essentials` should explicitly state all
```

File: kernel/linux/linux.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                    'file': './config',
+                    'rename': '.config',
+                },
+            ],
+        },
+    ],
+)
+def build(build):
+    packages = autotools.build(
+        configure=None,  # lambda: make('olddefconfig'),
```
Comment (doom) [inline kernel/linux/linux.py:61]: Should be removed

File: kernel/linux/linux.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        check=None,
+        compile=None,
+        install=install_linux,
+    )
+
+    # Drain documentation
+    # packages['kernel/linux-doc'].drain_build_cache('Documentation/*', 'usr/doc/linux/')
+
+    # Remove the dependencie of `kernel/linux-dev` to `kernel/linux`
```
Comment (doom) [inline kernel/linux/linux.py]:
```suggestion
    # Remove the dependency of `kernel/linux-dev` to `kernel/linux`
```

File: kernel/linux/linux.py [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+def build(build):
+    packages = autotools.build(
+        configure=None,  # lambda: make('olddefconfig'),
+        check=None,
+        compile=None,
+        install=install_linux,
+    )
+
+    # Drain documentation
+    # packages['kernel/linux-doc'].drain_build_cache('Documentation/*', 'usr/doc/linux/')
```
Comment (doom) [inline kernel/linux/linux.py:68]: Should probably be removed too

File: kernel/linux/linux.py (hunk 1 above)
Comment (Arignir) [reply kernel/linux/linux.py:25]: I left them on purpose.
This manifests only compiles `kernel/linux-dev` for now, not `kernel/linux`, and those lines are the one that we will uncomment to enable the later one.

**Pull request closed without merging**